src/components/uploadDocs/UploadInstructions.tsx

```diff
@@ -30,9 +31,8 @@ const CopyIdToken: React.FunctionComponent = () => {
     );
 };
 
-const AssayInstructions: React.FunctionComponent<IAssayInstructionsProps> = props => {
-    const assay = props.match.params.assay;
-    const path = `cidc-documentation/master/assays/${assay}.md`;
```

Comment:
We can no longer depend on the documentation living in the `assays/` subdirectory, so this component now takes a new prop `docPath` that should contain the path to the markdown document relative to the `cidc-documentation` repository root.